src/main/java/org/pac4j/dropwizard/Pac4jFactory.java

```diff
@@ -68,22 +70,54 @@
 
     private boolean sessionEnabled = true;
 
-    @NotNull
```

Comment:
Why remove these annotations? They are used by dropwizard to check the validity of the configuration :)